_includes/lesson_footer.html

```diff
@@ -40,12 +40,9 @@
 	<a href="{{ site.github.repository_url }}/blob/gh-pages/CITATION">Cite</a>
 	/
 	<a href="mailto:{{ site.email }}">Contact</a>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-md-12" align="center">
-      Using <a href="https://github.com/carpentries/styles/">The Carpentries style</a>
-      version <a href="https://github.com/carpentries/styles/releases/tag/v9.5.2">9.5.2</a>.
+	<br>
```

Comment:
I prefer to have this as `col-md-12` instead of under the other links as this is not part of the same "semantic" element.

Reply:
Although this is a different semantic element, it is not so important to be in the middle of a page. Perhaps, it could be combined with the license block.